src/components/Autocomplete.css

```diff
@@ -10,6 +10,8 @@
 
 .autocomplete .results-summary {
   margin: 10px;
+  direction: ltr;
```

Comment:
ahh - that's not necessary. We use `strings.json` and `debugger.properties` so that inside firefox we can load other properties files for RTL languages